src/lib/scroll/ScrollElement.js

```diff
@@ -114,8 +114,8 @@ class ScrollElement extends Component {
     })
   }
 
-  handleMouseLeave = e => {
-    this.props.onMouseLeave(e)
+  handleMouseLeave = () => {
+    // this.props.onMouseLeave(e)
```

Comment:
is there a reason for not deleting these?

Reply:
None. Will delete!